crates/apollo-router-core/src/error.rs

```diff
@@ -181,10 +181,12 @@ impl From<JoinError> for QueryPlannerError {
 }
 
 /// Error in the schema.
-#[derive(Debug, Error, Display)]
+#[derive(Debug, Error)]
 pub enum SchemaError {
     /// IO error: {0}
+    #[error("IO error: {0}")]
     IoError(#[from] std::io::Error),
     /// Parsing error(s).
+    #[error("parse errors {0:#?}")]
```

Comment:
Keep using Display as it provides the doc at the same time than the error message

Reply:
the parser errors don't expose any impl display, do you want me to build an impl display for ParseErrors ?

Reply:
no I meant displaydoc::Display

Reply:
But eventually parse errors should implement Display indeed. Just not in this repository

Reply:
Oh I see, thanks for clearing it up!